.github/workflows/build.yml

```diff
@@ -445,6 +451,8 @@ jobs:
         echo "PATH=/usr/lib/ccache:$PATH" >> $GITHUB_ENV
     - name: Configure ccache action
       uses: hendrikmuhs/ccache-action@v1.2
+      with:
+        save: ${{ github.event_name == 'push' }}
```

Comment:
Should we use a `max-size` here as well as in `.github/workflows/reusable-ubuntu.yml`?

(Same applies to `main`.)

Reply:
Oops! Yes, let me fix that in main first.